pages/public_cloud/containers_orchestration/managed_rancher_service/creating-kubernetes-pci-compute-instances/guide.en-gb.md

```diff
@@ -0,0 +1,178 @@
+> [!warning]
+>
+> You don't have to click the `Edit Auth Config`{.action} button.
+
+The OVHcloud Public Cloud credentials have been created. Now you can use them to create a Kubernetes cluster based on Public Cloud Instances (PCI). These credentials will be used to provision nodes in your clusters.
+You can use these credentials to create several Kubernetes clusters.
+
+### Creating a Kubernetes cluster on Compute Instances
+
+Log in to your Managed Rancher Service UI.
+
+![OVHcloud Managed Rancher Service UI](images/rancher-ui.png){.thumbnail}
+
+Click the `Create`{.action} button.
+
+![OVHcloud Managed Rancher Service Create Kubernetes PCI](images/rancher-create.png){.thumbnail}
+
+In Rancher you can create a Kubernetes cluster with different methods. To create a Kubernetes cluster running in Compute Instances, use the **Provision new nodes and create a cluster using RKE2/K3s** way and click the `OVHcloud Public Cloud`{.action} driver.
+
+Select an OVHcloud Public Cloud credential:
+
+![OVHcloud Managed Rancher Service Create Kubernetes PCI](images/rancher-select-creds.png){.thumbnail}
+
+Then, define the cluster name.
+
+![OVHcloud Managed Rancher Service Cluster Name](images/rancher-cluster-name.png){.thumbnail}
+
+In the **Machine Pools** section you will configure your cluster.
+
+> [!primary]
+>
+> When you configure a machine pool in Rancher, there are three roles that can be assigned to nodes: `etcd`, `Control Plane` and `Worker`.
+
+Here are some good practices:
+
+- At least 3 machines/nodes with the role `etcd` are needed to survive a loss of 1 node and have a minimum high availability configuration for etcd. 3 `etcd` nodes are generally sufficient for smaller and medium clusters, and 5 `etcd` nodes for large clusters.
+- At least 2 machines/nodes with the role `Control Plane` for master component high availability.
+- You can set both the `etcd` and `Control Plane` roles for one instance.
+- The `Worker` role should not be used or added to nodes with the `etcd` or `Control Plane` role.
+- At least 2 machines/nodes with the `Worker` role for workload rescheduling upon node failure.
+
+In this guide we will:
+
+- Create a machine pool with 3 compute instances for `etcd` and `Control Plane`.
+- Create a machine pool with 2 compute instances for `Worker`.
+
+For each of the machine pools, you have to:
+
+- Define the pool name (`node-pool-1` for example for the first machine pool).
+- Define machine count (3 for example for the first machine pool).
+- Select roles (check `etcd` and `Control Plane` for the first machine pool)/
+- Choose the region (`GRA9` for example for the first machine pool). If you want to check the availability of specific products that you plan to use alongside Kubernetes, you can refer to the [Availability of Public Cloud Product](/links/public-cloud/regions-pci) page.
+- Choose the flavor (`b2-7` for example for ). You can refer to the [OVHcloud Flavor list](https://www.ovhcloud.com/en-gb/public-cloud/prices/).
```

Comment:
@scraly, it seems some information is missing inside the () brackets. Can you please check?

Reply:
I fixed it, thanks :)